### DISCUSSION TITLE
認識編譯型語言&解釋型語言

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#402] [General]
上過老師的課後，應該都知道Python不用編譯...吧
不過似乎有點抽象...
以下網站解釋編譯型&解釋型的不同
[https://www.itread01.com/content/1542366250.html](url)
以下網站比較了解釋型語言使用的直譯器(interpreter)&編譯型語言使用的編譯器(compiler)
[https://www.gushiciku.cn/pl/ptE0/zh-tw](url)
感覺是寫得蠻詳細的啦
![image](https://img-blog.csdn.net/20180802084112768?watermark/2/text/aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3UwMTIxODQ1Mzk=/font/5a6L5L2T/fontsize/400/fill/I0JBQkFCMA==/dissolve/70)
![image](https://img-blog.csdn.net/20180802084124396?watermark/2/text/aHR0cHM6Ly9ibG9nLmNzZG4ubmV0L3UwMTIxODQ1Mzk=/font/5a6L5L2T/fontsize/400/fill/I0JBQkFCMA==/dissolve/70)
![image](https://mdimg.wxwenku.com/getimg/356ed03bdc643f9448b3f6485edc229bb87b80b3202fa9ebec36757e8790b1875c1eb5770be5835909f544534cea48c8.jpg)

**Comment by 11timyyy** [comment]
不像其他程式語言，python比較清楚簡潔又容易撰寫!

**Comment by chupochun** [comment]
原來python在其他程式語言中算是比較簡單的

**Comment by JunTingLin** [comment]
據我所知不管是直譯型(逐行轉換再輸出)還是編譯型(整個先compile再輸出)都會轉換成組合語言(ASM)再轉機器語言(0與1)
假如組語Assembly有興趣可以去修資工梁老師的組合語言!蠻好玩的

**Comment by ghost** [comment]
python真的簡單且直覺很多，但常常也被稱做傻瓜語言...

**Comment by pekoranoinu** [comment]
python用法簡單，但是速度卻比其他程式慢。

**Comment by Yuhsin000** [comment]
上課的題目裡有此名詞，感謝解答
